packages/cli/src/build/__tests__/other.ts

```diff
@@ -824,3 +824,101 @@ test(".d.ts", async () => {
 
   `);
 });
+
+test("server components", async () => {
+  const dir = await testdir({
+    "package.json": JSON.stringify({
+      name: "pkg",
+      main: "dist/pkg.cjs.js",
+      module: "dist/pkg.esm.js",
+    }),
+    "src/index.js": js`
+                      export { A } from "./client";
+                    `,
+    "src/client.js": js`
+                       "use client";
+                       export const A = "something";
+                     `,
+    "src/b.js": js`
+                  export const B = "b";
+                `,
+    "src/c.js": js`
+                  import { D } from "./d";
+                  export function C() {
+                    return D;
+                  }
+                `,
+    "src/d.js": js`
+                  "use client";
+                  export const D = "d";
+                `,
```

Comment:
those end up being unused - was this intentional?

Reply:
No, need to fix that, it was from debugging